### DISCUSSION TITLE
1) LLM response not labelled in Prodigy 2) dependency problems between prodigy and spacy-llm >= 0.6.0

### DISCUSSION THREAD

**Discussion by SilasFriby** [explosion/spacy-llm#360] [General]
Hi spacy-llm,

1) Based on the screen shot below, can you explain why "CEO" and "Despite the great performance in fiscal 2022, the global shipping industry is facing challenges in 2023," are not labelled although they appear in the response from GPT-4?

![Skærmbillede 2023-11-10 kl  14 12 29](https://github.com/explosion/spacy-llm/assets/25742618/f3503a90-79e2-4d27-b64a-a38698c659e9)

![Skærmbillede 2023-11-10 kl  14 13 20](https://github.com/explosion/spacy-llm/assets/25742618/30115d86-0045-4b99-8128-6dad96877c5c)

2) Is it not possible to use the latest version of spacy-llm together with prodigy?

Best regards
Silas

**Comment by rmitsch** [comment]
Hi @SilasFriby, thanks for reporting this.

> 2. Is it not possible to use the latest version of spacy-llm together with prodigy?

The Prodigy team is currently working on integrating support for the latest version of `spacy-llm`. Stay tuned!

> 1. Based on the screen shot below, can you explain why "CEO" and "Despite the great performance in fiscal 2022, the global shipping industry is facing challenges in 2023," are not labelled although they appear in the response from GPT-4?

This seems like a bug. It might be fixed by the latest version of `spacy-llm` that should be supported soon. In general: if you are facing issues with Prodigy, you are likely to receive more helpful replies on support.prodi.gy. The support team there will be able to determine whether the problem you are facing is a Prodigy or a `spacy-llm` issue.
